eq-author/src/components/CommentsPanel/index.test.js

```diff
@@ -452,4 +612,152 @@ describe("Comments Pane", () => {
     expect(newCommentSubscriptionWasCalled).toBeTruthy();
     expect(getByText("This is an edited comment")).toBeTruthy();
   });
+
+  ///////////////////////////////////////////Replies////////////////////////////////
```

Comment:
You can nest describe blocks within each other, so I would remove the comment and use another describe instead.